api/v1alpha1/nicfirmwaresource_types.go

```diff
@@ -27,6 +27,9 @@ type NicFirmwareSourceSpec struct {
 	// BFBUrlSource represents a url source for BlueField Bundle
 	// +optional
 	BFBUrlSource string `json:"bfbUrlSource,omitempty"`
+	// DocaSpcXCCUrlSource represents a url source for DOCA SPC-X CC .deb package for ubuntu 22.04
```

Comment:
I assume this API will be deprecated along the way.
Should we add some comment about it?